token-lending/cli/src/main.rs

```diff
@@ -797,10 +797,13 @@ fn command_create_lending_market(
         Some(&config.fee_payer.pubkey()),
     );
 
-    let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
+    let recent_blockhash = config.rpc_client.get_latest_blockhash()?;
```

Comment:
Changes in this file are to deprecate `get_recent_blockhash` in favor of `get_latest_blockhash` and `get_fee_for_message`, a result of upgrading to 1.9.5.